addons/udes_stock/tests/test_stock_picking.py

```diff
@@ -1882,3 +1882,79 @@ def test_priority_one_remains_on_cancelled_picking(self):
         """
         self.pick2.action_cancel()
         self.assertEqual(self.pick2.priority, "1")
+
+    def test_picking_can_be_deleted_in_draft(self):
+        """
+        Test that a picking can be deleted in draft state
+        """
+        pick = self.create_picking(
+            picking_type=self.picking_type_pick,
+            products_info=[{"product": self.fig, "uom_qty": 10}],
+            location_id=self.test_stock_location_02.id,
+            confirm=False,
+            assign=False,
+            priority="0",
```

Comment:
Q: Does priority matter?

Reply:
Dont think it matters, just an extra variable as priority is not a required field. But doesnt harm as well :)